src/unitxt/operators.py

```diff
@@ -211,7 +211,7 @@ def get_mapped_value(self, instance, key, mapper, val):
             return recursive_copy(mapper[val_as_str])
         if self.strict:
             raise KeyError(
-                f"value '{val}' in instance '{instance}' is not found in mapper '{mapper}', associated with field '{key}'."
+                f"value '{val}' in field '{key}' is not found in mapper '{mapper}'"
```

Comment:
`val_as_str` is actually the one missing from the mapper. I suggest to hint about this, in case users will use mappers from non-str (although we warn against this):

suggesting:  f"'{val_as_str}', the casted-to-string value from field '{key}', is not found in mapper '{mapper}'"